scripts/jerigon_zero_benchmark.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# ------------------------------------------------------------------------------
+set -exo pipefail
+
+# Args:
+# 1 --> Output file (Not used in the current script)
+
+# Get the number of processors for parallelism
+if [[ "$OSTYPE" == "darwin"* ]]; then
+    num_procs=$(sysctl -n hw.physicalcpu)
+else
+    num_procs=$(nproc)
+fi
+
+# Force the working directory to always be the repository root.
+REPO_ROOT=$(git rev-parse --show-toplevel)
+PROOF_OUTPUT_DIR="${REPO_ROOT}/proofs"
+BLOCK_BATCH_SIZE="${BLOCK_BATCH_SIZE:-8}"
+
+# Logging setup
+OUTPUT_LOG="jerigon_zero_output.log"
+BLOCK_OUTPUT_LOG="jerigon_zero_block_output.log"
+PROOFS_FILE_LIST="${PROOF_OUTPUT_DIR}/proof_files.json"
+
+# Ensure necessary directories exist
+mkdir -p "$PROOF_OUTPUT_DIR"
+
+# Set environment variables for parallelism and logging
+export RAYON_NUM_THREADS=$num_procs
+export TOKIO_WORKER_THREADS=$num_procs
+export RUST_MIN_STACK=33554432
+export RUST_BACKTRACE=full
+export RUST_LOG=info
+
+# Log the current date and time
+echo "$(date +"%Y-%m-%d %H:%M:%S")" &>> "$OUTPUT_LOG"
+
+# Define the blocks to process
+blocks=(100 200 300 400 500)
```

Comment:
@praetoriansentry @Nashtare currently using random blocks, any idea which blocks to use for benchmark testing

Reply:
Maybe we could read the list from the config file (CSV to make it simple?).

Reply:
yes this is just a placeholder. we will make choose block numbers first and if thats too much, we will put that in config file

Reply:
we are here testing multiple blocks from test data
701 is using just 1 witness files. 701 can be extended for multiple specific witnesses. i remember einar mentioning that he needs some cron job where he can just put in a witness file for some specific cases and make sure they never fail in future.